pkg/amazonlinux2016.09/Dockerfile

```diff
@@ -44,35 +22,23 @@ RUN mkdir -p /etc/hubble/hubble.d /opt/hubble /opt/osquery /var/log/hubble_osque
 #osquery build start
 #osquery should be built first since requirements for other packages can interfere with osquery dependencies
 #to build, osquery scripts want sudo and a user to sudo with.
+ENV OSQUERY_BUILD_USER=osquerybuilder
+ARG BUILD_OSQUERY=true
+ARG BUILD_OSQUERY_LOCALLY=false
+ARG OSQUERY_SRC_VERSION=osquery_3.3.2_tob
+COPY osquery /osquery
+ENV OSQUERY_TAR_FILENAME=$OSQUERY_SRC_VERSION"_amazonlinux2016.09.tar"
+COPY $OSQUERY_TAR_FILENAME /
```

Comment:
This is to save the step of extracting. Since this tar is to be extracted only when pre-built osquery binary is being used, extracting it always gives us no benefit.